oracle/blockeligibilityvalidator.go

```diff
@@ -31,11 +34,33 @@ func NewBlockEligibilityValidator(committeeSize int32, layersPerEpoch uint16, ac
 	}
 }
 
-func (v BlockEligibilityValidator) BlockEligible(block *types.BlockHeader) (bool, error) {
+func (v BlockEligibilityValidator) BlockEligible(block *types.Block) (bool, error) {
```

Comment:
Consider renaming to `BlockSignedAndEligible` (or something else that reflects that this is the place where we validate the miner's signature and not only the block's eligibility